app/index.html

```diff
@@ -325,6 +325,7 @@ <h4 class="modal-title" id="gcc-error-label" data-bind="html: errorWarningLabel"
         <!-- build:js scripts/vendor.js -->
         <!-- bower:js -->
         <script src="../bower_components/jquery/dist/jquery.js"></script>
+        <script src="../bower_components/bootstrap-sass-official/assets/javascripts/bootstrap.js"></script>
```

Comment:
Probably added by `bowerInstall` when watching for changes. Its okay, I'll remove those myself once this is merged into `master`.